packages/data/src/ethers.ts

```diff
@@ -295,4 +295,91 @@ export default class SemaphoreEthers {
 
         return this._contract.hasMember(groupId, member)
     }
+
+    /**
+     * Listens to the GroupCreated event.
+     * @param callback Called with the groupId of the newly created group.
+     */
+    onGroup(callback: (groupId: string) => void): void {
+        this._contract.on("GroupCreated", (groupId: string) => {
```

Comment:
```suggestion
        this._contract.on("GroupCreated", (groupId) => {
```

We could remove the string type here.

Also, what about adding the additional `event` parameter to get additional event info? See the `Listening to Events` section here: https://docs.ethers.org/v6/getting-started/#starting-contracts

Reply:
Done!